CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ Important changes to data-models, configuration and migrations between each
 AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
+ * Bumped runtimeVersion to `2020.11.20`.
+ * Upgraded Dart analysis SDK to `2.12.0-51.0.dev`.
```

Comment:
Why not go for the beta version?

Reply:
It feels like that dev will be ahead of beta with a reasonable lead, giving us ample time for the upgrades. People closely following the beta will always face the issue that we are several days behind. I hope that going with dev gives us just enough time to catch up.